tests/dev_wrappers/test_lambda_actions/test_clip_actions.py

```diff
@@ -0,0 +1,287 @@
+"""Test suite for ClipActionsV0."""
+from collections import OrderedDict
+
+import numpy as np
+import pytest
+
+import gym
+from tests.dev_wrappers.mock_data import (
+    DICT_SPACE,
+    DICT_WITHIN_TUPLE_SPACE,
+    DOUBLY_NESTED_DICT_SPACE,
+    DOUBLY_NESTED_TUPLE_SPACE,
+    NESTED_DICT_SPACE,
+    NESTED_TUPLE_SPACE,
+    NEW_BOX_HIGH,
+    NEW_BOX_LOW,
+    NUM_ENVS,
+    SEED,
+    TUPLE_SPACE,
+    TUPLE_WITHIN_DICT_SPACE,
+)
+from tests.dev_wrappers.utils import TestingEnv
+
+try:
```

Comment:
`pytest.importorskip` allows you to try import a module otherwise the file is skipped

Reply:
With importorskip I can do this at module level:

```python
wrappers = pytest.importorskip("gym.wrappers")
```

But this does not seems to work:
```python
ClipActionsV0 = pytest.importorskip("gym.wrappers.ClipActionsV0")
```  

Another option I can think of is to do like this:  

``` python
pytest.importorskip("jumpy")
from gym.wrappers import ClipActionsV0 
```
The only caveat is that I need to add `# noqa` to the import because of pre-commit, what do you think it is better?